### DISCUSSION TITLE
`Ctrl+R` Shortcut to reload/re-index current directory

### DISCUSSION THREAD

**Discussion by Crilum** [spacedriveapp/spacedrive#1595] [Feature Request]
Most file managers will reload the current directory when `Ctrl+R` is pressed. 

I use this a lot, and I think this would help other people too.

**Comment by brxken128** [comment]
We plan to implement this, the reload button should spawn the shallow indexer but I'm not sure it currently does (nor does it provide any visual feedback). I'll make sure the button works in time for the next release, as well as a toast, and link the shortcut to it.

In dev mode, CMD+R or CTRL+R will currently reload the actual webview itself but I think we should change that shortcut to avoid confusion between dev and production builds (this functionality is disabled in production builds anyway), so it'd have two clashing functions assigned to one relatively common keybind.
